apps/ui/.env.local.example

```
@@ -77,3 +77,8 @@ APP_PUBLIC_URL=http://localhost:3000
 
 # Strapi Preview Shared Secret
 STRAPI_PREVIEW_SECRET=your-secret-key
+
+# -------------------------- imgproxy ---------------------------------
+
+# Base URL of a self-hosted imgproxy instance (enables on-the-fly image resizing & retina support)
+# IMGPROXY_URL=https://imgproxy.example.com
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)